docs-2.0/1.introduction/2.1.path.md

```diff
@@ -0,0 +1,47 @@
+# Path types
+
+In graph theory, a path in a graph is a finite or infinite sequence of edges which joins a sequence of vertices. Paths are fundamental concepts of graph theory.
+
+Paths can be categorized into 3 types: `walk`, `trail`, and `path`. For more information, see [wikipedia](https://en.wikipedia.org/wiki/Path_(graph_theory)#Walk,_trail,_path).
+
+The following picture is an example for a brief introduction.
+
+![path graph](../20.appendix/path.png)
+
+## walk
+
+A `walk` is a finite or infinite sequence of edges. Both vertices and edges can be repeatedly visited in graph traversal.
+
+In the above path graph, C, D, and E form a cycle. So, this path graph contains infinite paths, such as `A->B->C->D->E`, `A->B->C->D->E->C`, and `A->B->C->D->E->C->D`.
+
+!!! note
+
+    `GO` statements use `walk`.
+
+## trail
+
+A `trail` is a finite sequence of edges. Only vertices can be repeatedly visited in graph traversal. The Seven Bridges of Königsberg is a typical `trail`.
+
+In the above path graph, edges cannot be repeatedly visited. So, this path graph contains finite paths. The longest path in this graph consists of 5 edges: `A->B->C->D->E->C`.
```

Comment:
There is no such thing as a path graph. Let's make it clear:

* A graph is a graph in graph theory. Graphs have vertices and edges.
* A picture or figure is, well, something like an image or a drawing.

So, maybe we should use "In the above picture/figure"? WDYT?